### ISSUE TITLE
[Programmatic Access – Make Code - Publish project] Incorrect role is defined for 'Embed' button control present under 'Share project' pop-up.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3596]
**User Experience:**
Users who depends upon screen reader get confused if the Screen reader reads incorrect role of any control. They'll not be able to know whether it is a link or a button or whether it is a non-interactive control. Users will understand it to be a interactive control as screen reader announces link while navigating in scan mode and hence it will not be a good usable experience for the users. 

**Note:** User credentials should NOT be included in the bug. 

**Pre-Requisite:**
Open F12 Developer tool.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Share" option appearing in header section and select it.
6. Navigate to "Publish project" under  "Share Project" dialog and select it.
7. Navigate to all the control appearing like QR Code, Embed(Code, Editor and Simulator).
8. Verify that incorrect role is defined for 'Embed' button.

**Actual Result:**
Incorrect role 'Link' is defined for 'Embed' button control present under 'Share project' pop-up.

**Expected Result:**
Correct role should be define as 'Button' instead of link for 'Embed' button control present under 'Share project' pop-up.

**Note:**
Same issue exists on Firefox + NVDA & Chrome + JAWS.

**WCAG Reference:**
[WCAG 4.1.2 – Name, Role, Value ](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG4.1.2, #Win10-Edge(Chromium)

![WCAG4 1 2_Publish project_Incorrect role](https://user-images.githubusercontent.com/72871089/98230046-b2bdf300-1f80-11eb-8c09-2acca3cade34.jpg)

**Comment by livcheerful** [comment]
fixed with https://github.com/microsoft/pxt/pull/8062/files#diff-3e7cafbd95631be2faaeb5d32fe7bfc6b5d33f707af4d40d2067ac479d7953c9R411
